demos/api_demo.py

```diff
@@ -257,6 +257,11 @@
                  params={'websocket': websocket})
 print(r.text)
 
+print('/api/derive/session/progress')
```

Comment:
Kindly move this before `/api/derive/staking/validators`, to keep things in alphabetical order